Google.PowerShell/BigQuery/BqTabledata.cs

```diff
@@ -0,0 +1,233 @@
+// Copyright 2015-2017 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Cloud.BigQuery.V2;
+using Google.Apis.Bigquery.v2.Data;
+using Google.PowerShell.Common;
+using System;
+using System.Net;
+using System.Linq;
+using System.Management.Automation;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// Possible types for each column in a TableSchema
+    /// </summary>
+    public enum ColumnType
+    {
+        STRING, BYTES, INTEGER, INT64,
+        FLOAT, FLOAT64, BOOLEAN, BOOL,
+        TIMESTAMP, DATE, TIME, DATETIME,
+        RECORD, STRUCT
+    }
+
+    /// <summary>
+    /// Possible types for each column in a TableSchema
+    /// </summary>
+    public enum ColumnMode
+    {
+        NULLABLE, REQUIRED, REPEATED
+    }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Instantiates a new BQ schema or adds a field to a pre-existing schema.
+    /// </para>
+    /// <para type="description">
+    /// This command defines one column of a TableSchema. To create a multi-row schema, chain 
+    /// multiple instances of this command together on the pipeline. Required fields for each 
+    /// column are Name and Type. Possible values for Type include STRING, BYTES, INTEGER, 
+    /// FLOAT, BOOLEAN, TIMESTAMP, DATE, TIME, DATETIME, and RECORD (where RECORD indicates 
+    /// that the field contains a nested schema). Case is ignored for both Type and Mode. 
+    /// This command forwards all TableFieldSchemas that it is passed, and will add a new 
+    /// TableFieldSchema object to the pipeline.
+    /// </para>
+    /// <example>
+    ///   <code>
+    /// PS C:\> $table = $dataset_books | New-BqTable "book_info"
+    /// PS C:\> $result = New-BqSchema -Name "Author" -Type "STRING" | `
```

Comment:
FYI, ending a line with a pipe is one of the few way to indicate the statement continues on the next line without needing a backtick.